wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,392 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+<!-- Maintainers Note: Please refer to the original pull request for more info on sources at https://github.com/ppy/osu-wiki/pull/4720 -->
+
+# osu! rankings history
+
+The following is a list of players who are recognised to have reached Rank #1 in the [osu! global rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years, the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (6 October 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list only includes players who have reached Rank #1 on the main leaderboard that was used at the time. *Italicized* dates and numbers indicate that the corresponding player is continuing their reign as osu!'s Rank #1 player either from the previous year or into the following year.
+
+While it has been assumed that ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) was the first Rank #1 player, no sources have been able to validate this claim.
```

Comment:
```suggestion
While ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) was assumed to be the first rank #1 holder, no sources were able to validate this claim.
```

Reply:
While changing to `have been` to `were` probably doesn't change the meaning too much, the thing that we wanted to try and convey is that it's not that no sources exist that could prove this, but that we weren't able to find any, there could be something out there that can prove it, but we are yet to find it.

Reply:
Resolved with https://github.com/ppy/osu-wiki/pull/4720/commits/b9d74eb7bc8685b477ba43c929328ba7a27275cd